packages/neuron-ui/src/components/NervosDAO/index.tsx

```diff
@@ -179,6 +179,7 @@ const NervosDAO = () => {
                 genesisBlockTimestamp,
                 connectionStatus,
                 isPrivacyMode,
+                hasCkbBalance: !!wallet.balance && wallet.balance !== '0',
```

Comment:
Why not use `+wallet.balance > 0`

Reply:
The `wallet.balance` should transform to BigInt, but for comparison to 0, loss of precision doesn't matter.
I have changed it to `+wallet.balance > 0`